setup.cfg

```diff
@@ -0,0 +1,85 @@
+[metadata]
+author = SciTools Developers
+author_email = scitools-iris-dev@googlegroups.com
```

Comment:
It would be great if we had an alternate email address...

Reply:
SciTools owns the https://scitools.org.uk/ domain. Domains almost always have associated email addresses?

Reply:
Yeah, dunno... I'm just thinking that if we bale on Google Groups (I hope :crossed_fingers:), then we'd need to tee up an alternate email address

Reply:
@tkknight Do you know the answer to @trexfeathers's question?

Reply:
> if we bale on Google Groups (I hope 🤞), then we'd need to tee up an alternate email address
It could be that email is just the "old way", and browser-based discussions is the new.
Could we therfore omit 'author_email' and just rely on the 'project_urls' (with 'discussions' sub-heading) ?

Reply:
>It could be that email is just the "old way", and browser-based discussions is the new.
Could we therfore omit 'author_email' and just rely on the 'project_urls' (with its 'Discussions' and 'Issues' sub-headings) ?

I'm interested in what you think @bjlittle.

Reply:
I'm not aware of having a domain related email address for scitools.  Not sure who originally setup up the domain, they may know.

Reply:
I don't think this is a blocker to merging this PR.

I've done a minimal search to see if I can find a setuptools/PyPI metadata best practice, and not found anything obvious.
Alternatively, I turned to the wisdom of the crowd and sampled several PyPI projects, all of which provided an `author_email`.
Therefore, I'd be keen for us to maintain this and not omit it.